changelog.d/3459.feature

```diff
@@ -0,0 +1 @@
+Refuse to serve content from users who have erased their content
```

Comment:
This is kinda nonsensical: if the content is erased, how can we serve it? 

Also: what does "content"  mean, and what does it mean when it is "from" a user?